app/helpers/formatting_helper.rb

```diff
@@ -23,19 +25,27 @@ def rss_status_content_format(status)
 
     before_html = begin
       if status.spoiler_text?
-        "<p><strong>#{I18n.t('rss.content_warning', locale: available_locale_or_nil(status.language) || I18n.default_locale)}</strong> #{h(status.spoiler_text)}</p><hr />"
-      else
-        ''
+        tag.p do
```

Comment:
I wasn't sure of the preferred formatting for these moderately complicated blocks so please do let me know if e.g. things should be broken out into separate local variables/methods or any formatting preferences (rubocop seemed happy).